### PR TITLE
chore: P-1044 merge all identity tests into one

### PR DESCRIPTION
**Pull request by wli-pro**
### Context
As title

### Labels

Please apply following PR-related labels when appropriate:
- `C0-breaking`: if your change could break the existing client, e.g. API change, critical logic change 
- `C1-noteworthy`: if your change is non-breaking, but is still worth noticing for the client, e.g. reference code improvement

### Testing Evidences

Please attach any relevant evidences if applicable

### CODE REVIEW

**Review by jonalvarezz — APPROVED** [review]
having all the tests in a single place makes sense, thanks @higherordertech 

Because of how ts-tests were created, I understand we need to clean up storage with `clear_linked_identities`. I guess it is ok to have development-only features, but I'd have preferred to re-work ts-tests to for example use random accounts each time instead of requiring a cleanup on every run. I understand it'd be out of scope, too.

**Comment by Kailai-Wang** [comment]
Thanks!

It's good to go once CI passes - please note we expect the tests to be adjusted again to align with the omni-account usage (very soon, but the effort should not that huge)

**Comment by wli-pro** [comment]
> having all the tests in a single place makes sense, thanks @higherordertech
> 
> Because of how ts-tests were created, I understand we need to clean up storage with `clear_linked_identities`. I guess it is ok to have development-only features, but I'd have preferred to re-work ts-tests to for example use random accounts each time instead of requiring a cleanup on every run. I understand it'd be out of scope, too.

Spot on, I updated the tests to not rely on cli to clear idgraph but just randomize new account for each test case now.

File: tee-worker/identity/ts-tests/integration-tests/common/helpers.ts [hunk 1]
```diff
@@ -115,3 +130,16 @@ export function stfErrorToString(stfError: StfError): string {
 
     return stfError.type;
 }
+
+export async function cleanIDGraphs(context: IntegrationTestContext, teeShieldingKey: KeyObject) {
```
Comment (kziemianek) [inline tee-worker/identity/ts-tests/integration-tests/common/helpers.ts]: It looks like it's not used anymore, can it be removed ?

File: tee-worker/identity/ts-tests/integration-tests/common/di-utils.ts [hunk 2]
```diff
@@ -347,6 +347,26 @@ export async function createSignedTrustedGetterIdGraph(
     return parachainApi.createType('Getter', { trusted: getterSigned });
 }
 
+export async function createSignedTrustedCallCleanIDGraphs(
```
Comment (kziemianek) [inline tee-worker/identity/ts-tests/integration-tests/common/di-utils.ts]: Remove if not used ?

File: tee-worker/identity/ts-tests/integration-tests/common/utils/identity-helper.ts [hunk 3]
```diff
@@ -146,15 +146,15 @@ export async function buildValidations(
     signerIdentitity: CorePrimitivesIdentity,
     linkIdentity: CorePrimitivesIdentity,
     startingSidechainNonce: number,
-    network: 'ethereum' | 'substrate' | 'bitcoin' | 'solana',
+    network: 'evm' | 'ethereum' | 'substrate' | 'bitcoin' | 'solana',
```
Comment (kziemianek) [inline tee-worker/identity/ts-tests/integration-tests/common/utils/identity-helper.ts]: Can 'ethereum' be removed ?

**Review by kziemianek — APPROVED** [review]

**Pull request merged**